.github/workflows/tests.yml

```diff
@@ -0,0 +1,24 @@
+name: tests for project management statistics
+on:
+  - push
+  - pull_request
+
+jobs:
+  pms:
+    name: project management statistics
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.10']
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install requirements
```

Comment:
```
Run sudo apt-get install -y build-essential
Reading package lists...
Building dependency tree...
Reading state information...
build-essential is already the newest version ([12](https://github.com/ilausuch/project_management_statistics/actions/runs/4084523557/jobs/7041323650#step:4:13).9ubuntu3).
0 upgraded, 0 newly installed, 0 to remove and [13](https://github.com/ilausuch/project_management_statistics/actions/runs/4084523557/jobs/7041323650#step:4:14) not upgraded.
```

Reply:
check executed job you don't need to install this

Reply:
https://github.com/ilausuch/project_management_statistics/actions/runs/4084523557/jobs/7041323650

Reply:
Removed :)